IdHTTPWebsocketClient.pas

```diff
@@ -160,7 +160,10 @@ implementation
   Winapi.Windows,
 {$ENDIF}
 {$IF DEFINED(POSIX)}
-  Posix.SysSocket, Posix.Fcntl, FMX.Platform,
+  Posix.SysSocket, Posix.Fcntl,
+{$IFDEF DEFINED(ANDROID) or DEFINED(MACOS)}
```

Comment:
This is incorrect.

{$IFDEF DEFINED(ANDROID)... } should be
{$IF DEFINED(ANDROID)...}